VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/config/E2EConfig.java

```diff
@@ -0,0 +1,6 @@
+package org.edx.mobile.test.feature.config;
+
+public class E2EConfig {
+    public static String EMAIL_TEMPLATE = "test-{unique_id}@example.com";
```

Comment:
Can you use what's here https://openedx.atlassian.net/wiki/display/MA/App+Testing+Accounts to setup test accounts?